dotnet/samples/Demos/AIModelRouter/CustomRouter.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+#pragma warning disable SKEXP0001
+#pragma warning disable SKEXP0010
+#pragma warning disable CA2249 // Consider using 'string.Contains' instead of 'string.IndexOf'
+
+namespace AIModelRouter;
+
+/// <summary>
+/// This class is for demonstration purposes only.
+/// In a real-world scenario, you would use a more sophisticated routing mechanism, such as another local model for
+/// deciding which service to use based on the user's input or any other criteria.
+/// </summary>
+public class CustomRouter()
+{
+    /// <summary>
+    /// Returns the best service id to use based on the user's input.
+    /// This demonstration uses a simple logic where your input is checked for specific keywords as a deciding factor,
+    /// if no keyword is found it defaults to the first service in the list.
+    /// </summary>
+    /// <param name="lookupPrompt">User's input prompt</param>
+    /// <param name="serviceIds">List of service ids to choose from in order of importance, defaulting to the first</param>
+    /// <returns>Service id.</returns>
+    public string FindService(string lookupPrompt, IReadOnlyList<string> serviceIds)
+    {
+        // The order matters, if the keyword is not found, the last one is used.
```

Comment:
The comment says `last one` but we return first service, should we update it?
```suggestion
        // The order matters, if the keyword is not found, the first one is used.
```